coderd/database/queries/groups.sql

```diff
@@ -70,15 +74,15 @@ INSERT INTO groups (
 	id,
 	name,
 	organization_id,
-    	    	source
+						source
 )
 SELECT
-    	    	gen_random_uuid(),
-    	    	group_name,
-    	    	@organization_id,
-    	    	@source
+						gen_random_uuid(),
+						group_name,
+						@organization_id,
+						@source
 FROM
-    	    	UNNEST(@group_names :: text[]) AS group_name
+						UNNEST(@group_names :: text[]) AS group_name
 -- If the name conflicts, do nothing.
 ON CONFLICT DO NOTHING
 RETURNING *;
```

Comment:
Tabs vs spaces is so annoying in our sql files. We are inconsistent I know :cry: